package.json

```diff
@@ -10,7 +10,7 @@
   },
   "bin": {
     "generate-subsection": "./index.js",
-    "generate-ionic-subsection": "./index.js --template-type ionic"
+    "generate-ionic-subsection": "./ionic-index.js"
```

Comment:
🤞 This worked with local testing but I had to manually add the command to the `node_modules` `.bin` directory because you don't go through the install process when you use yarn "links".